bin/origami-image-set-tools.js

```diff
@@ -48,6 +48,24 @@ program
 		});
 	});
 
+// Command to purge images from Origami Image Service
+program
+	.command('purge')
+	.option('-s, --source-directory <dir>', 'The directory to look for source images in', process.env.IMAGESET_SOURCE_DIRECTORY)
+	.option('-c, --scheme <scheme>', 'The custom scheme to purge this image set under', process.env.IMAGESET_SCHEME)
+	.option('-v, --scheme-version <version>', 'The version to publish this image set under', process.env.IMAGESET_VERSION)
```

Comment:
I don't think `--scheme-version` is used by this script

Reply:
It is used, I forgot to pass it through to the method 🙈